ftp-upload/src/lambda.ts

```diff
@@ -1,13 +1,14 @@
 import {Config} from './config';
-import {Readable} from "stream";
 
 const AWS = require('aws-sdk');
-const ftp = require('ftp');
+const FtpClient = require('ftp');
```

Comment:
The 'ftp' required is exposed by `@icetee/ftp` dependency I presume? By this I mean that the `ftp` reference is still valid to `require` as-is?

Reply:
Good point!

Reply:
The build didn't complain but I prefer to avoid any ambiguity.